src/styles/AddBook.module.css

```diff
@@ -1,5 +1,6 @@
 .container h3 {
-  color: #888;
+  padding: 6px;
+  color: #747474;
 
   /* padding-left: 140px; */
```

Comment:
- [ ] As I mentioned above :point_up:  please don't push commented code because it's your codebase messy if your want to check your previous code then check your GitHub commit history